src/api/constants.ts

```diff
@@ -3,7 +3,15 @@ const API_V1 = 'api/v1';
 // Streams Management
 export const LOG_STREAM_LIST_URL = `${API_V1}/logstream`;
 export const LOG_STREAMS_SCHEMA_URL = (streamName: string) => `${LOG_STREAM_LIST_URL}/${streamName}/schema`;
-export const LOG_QUERY_URL = `${API_V1}/query`;
+export const LOG_QUERY_URL = (params?: Record<string, string>) => {
+	if (params) {
+		for (let i in params) {
```

Comment:
As mentioned there should be a separate parser function.
What if I want a query string for some other endpoint?